spec.bs

```diff
@@ -463,7 +465,8 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
     with |winner|'s [=generated bid/ad descriptor=]. TODO: resolve |p| with urn-uuid, instead of a
     URL.
   1. Run [=generate reporting URLs=] with |auctionConfig| and |winner|.
-  1. TODO: UpdateInterestGroupsPostAuction.

```

Comment:
I think this TODO was meant to cover other things, like updating `bidCount` and `prevWins`.  Can we add back something like "TODO: Update bidCount and prevWins for interest groups that participated in the auction."